src/Compiler/Driver/CompilerConfig.fs

```diff
@@ -47,6 +47,9 @@ let FSharpScriptFileSuffixes = [ ".fsscript"; ".fsx" ]
 let FSharpIndentationAwareSyntaxFileSuffixes =
     [ ".fs"; ".fsscript"; ".fsx"; ".fsi" ]
 
+let FsharpExperimentalFeaturesEnabledAutomatically =
```

Comment:
I would like to start this pattern for command line switches which we would like to dogfood.
"FSHARP_EXPERIMENTAL_FEATURES"

I.e. not relying on us to remember to turn cmd switches on/off, but rather having this EnvVar set for each compiler developer and using it for all features we would like to have "Eventually enabled for everyone, just needs more testing".